db/migrate/20251126005628_enable_variant_tag_and_enable_inventory_for_enterprise_using_it.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+class EnableVariantTagAndEnableInventoryForEnterpriseUsingIt < ActiveRecord::Migration[7.1]
+  def up
+    Flipper.disable_group(:variant_tag, "enterprise_created_after_2025_08_11")
```

Comment:
Why are we disabling variant tags for new enterprises? I thought we want to enable it?
I guess this gets overridden below by `Flipper.enable_group(:variant_tag, "enterprise_with_no_inventory")`, but I still don't see the point of this line.